resource-managers/kubernetes/docker/src/main/dockerfiles/spark/entrypoint.sh

```diff
@@ -79,6 +79,14 @@ case "$1" in
     ;;
   executor)
     shift 1
+    # If the execid is set to the generic EXECID parse the pod name
+    if [ "${SPARK_EXECUTOR_ID}" == "EXECID" ]; then
```

Comment:
Instead of relying on pod-name, are there other options that you have explored, like having the driver mint out executor ids upon registration?

Reply:
That didn't seem to be a good match since our executor snapshot depends on being able to determine the exec id for a given pod as well and if we did it fully dynamically we wouldn't be able to know and we'd have to make even bigger changes to how we handle snapshot updates.

Reply:
So we had a call about this, and we'll handle it by mutating the labels on the pods.